school-library/classroom.rb

```diff
@@ -0,0 +1,12 @@
+class Classroom
+  attr_accessor :label, :student
+
+  def initialize(label)
+    @label = label
+    @student = []
+  end
+
+  def add_student
+    @student.push(student)
+  end
```

Comment:
- [ ] I think we're missing part of the association here. We know that a classroom `has many` students, and that's the association made here. But we're missing the part where a student `belongs to` a classroom. 
 
- [ ] Then on the `student` class, we should have a `classromm=` method as well, where we can add the student to a classroom and set its own classroom.

Reply:
What I meant by this is we want to add the student, to the array of students of the classroom, but we also want to declare on the student itself the classroom where he belongs to, so something like this:

```
def add_student(student)
  @students.push(student) unless @students.include?(student)
  student.classroom = self
end
```

Since most of the relationships are two ways, we must make sure to create references to each other in both instances of the class, the difference is sometimes it's going to be an array, others just the single reference: `has many`, `has one`, `belongs to`, etc.